packages/myst-cli/src/process/mdast.ts

```diff
@@ -309,7 +320,7 @@ export async function postProcessMdast(
   });
   await pipe.run(mdast, vfile);
 
-  // Ensure there are keys on every node after post processing
+  // Ensure there are keys on every node after post-processing
```

Comment:
```suggestion
  // Ensure there are keys on every node after post processing
```

Revert comment change